climt/_core/initialization.py

```diff
@@ -194,7 +206,7 @@ def __init__(
                 "Argument mid_or_interface_levels must be 'mid' or 'interface'")
         self.diagnostic_properties = {
             output_name: {
-                'dims': ['*'],
+                'dims': [vertical_dimension],
```

Comment:
This should be the same dimensionality as air_pressure. In other words, `['mid_levels', '*']`.